.github/workflows/CI.yml

```diff
@@ -22,9 +22,13 @@ jobs:
        run: |
         python -m pip install --upgrade pip
         pip install flake8
+        pip install isort
```

Comment:
While we are doing this, seems like we should have a root Pipfile that indicates which version of these tools we are using, what do you think?

Reply:
Created a JIRA issue: [Create a root pipfile for python stuff](https://opensearch.atlassian.net/browse/MIGRATIONS-1848)